docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
+| Version   | Path														|
+| --------- | --------------------------------------------------------	|
+| >= 4.9.0  | conditional/namespaces/{namespace}/imagestreams/{name}.json |
+
+### Config ID
+`conditional/image_streams_of_namespace`
 
-API reference:
+### Released version
+- 4.9.0
 
-	https://docs.openshift.com/container-platform/4.7/rest_api/image_apis/imagestream-image-openshift-io-v1.html#apisimage-openshift-iov1namespacesnamespaceimagestreams
+### Backported versions
+None
 
-* Location in archive: conditional/namespaces/{namespace}/imagestreams/{name}
-* Id in config: conditional/image_streams_of_namespace
-* Since versions:
-  - 4.9+
+### Changes
+None
 
 
 ## InstallPlans
 
-collects Top x InstallPlans from all openshift namespaces.
-Because InstallPlans have unique generated names, it groups them by namespace and the "template"
-for name generation from field generateName.
+Collects top X InstallPlans from all openshift namespaces. Because InstallPlans have
+unique generated names, it groups them by namespace and the "template" for name generation from field generateName.
 It also collects Total number of all installplans and all non-unique installplans.
 
-The Operators-Framework api https://github.com/operator-framework/api/blob/master/pkg/operators/v1alpha1/installplan_types.go#L26
+### API Reference
+- https://github.com/operator-framework/api/blob/master/pkg/operators/v1alpha1/installplan_types.go#L26
 
-* Location in archive: config/installplans/
-* Id in config: clusterconfig/install_plans
-* Since versions:
-  - 4.5.33+
-  - 4.6.16+
-  - 4.7+
+### Sample data
+- [docs/insights-archive-sample/config/instalplans.json](./insights-archive-sample/config/instalplans.json)
+
+### Location in archive
+- `config/instalplans.json`
+
+### Config ID
+`clusterconfig/install_plans`
+
+### Released version
+- 4.7.0
+
+### Backported versions
+- 4.5.33+
+- 4.6.16+
+
+### Changes
+None
 
 
 ## JaegerCR
 
-collects maximum of 5 jaegers.jaegertracing.io custom resources
-installed in the cluster
+Collects maximum of 5 `jaegers.jaegertracing.io` custom resources installed in the cluster.
+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/config/jaegertracing.io/jaeger1.json](./insights-archive-sample/config/jaegertracing.io/jaeger1.json)
+
+### Location in archive
+- `config/jaegers.jaegertracing.io/{name}.json`
+
+### Config ID
+`clusterconfig/jaegers`
 
-* Location in archive: config/jaegertracing.io/
-* Id in config: clusterconfig/jaegers
-* Since versions:
-  - 4.10+
+### Released version
+- 4.10.0
+
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## KubeControllerManagerLogs
 
-collects logs from kube-controller-manager pods in the openshift-kube-controller-manager namespace with following substrings:
-  - "Internal error occurred: error resolving resource",
-  - "syncing garbage collector with updated resources from discovery",
+Collects ogs from kube-controller-manager pods in the
```

Comment:
Little typo here. I would also suggest to use the same formatting for the pod name as for the namespace.

```suggestion
Collects logs from `kube-controller-manager` pods in the
```